fs/expose/dokan/__init__.py

```diff
@@ -5,19 +5,17 @@
 Expose an FS object to the native filesystem via Dokan.
 
 This module provides the necessary interfaces to mount an FS object into
-the local filesystem using Dokan on win32::
+the local filesystem using Dokan on win32:
 
-    http://dokan-dev.net/en/
+    http://dokan-dev.github.io/
 
-For simple usage, the function 'mount' takes an FS object and a drive letter,
-and exposes the given FS as that drive::
+For simple usage, the function 'mount' takes an FS object and path,
+and exposes the given FS as that path::
```

Comment:
Should the path already exist, or not? I'm guessing the `Q:\` drive in the example below didn't exist (?), but does mounting to an arbitrary path require the path to exist first (and if so, does it need to be an empty folder?) or will Dokan somehow create the new path?